docs/guides/latest/volumesnapshot/deployment-backup.md

```diff
@@ -0,0 +1,262 @@
+
+* `spec.target.ref`  refers to the backup target. `apiVersion`, `kind` and `name` refers to the `apiVersion`, `kind` and `name` of the targeted workload respectively. Stash will use this information to create a Volume Snapshotter Job for Creating Volume Snapshot.
+
+* `spec.target.snapshotClassName` indicates the [VolumeSnapshotClass](https://kubernetes.io/docs/concepts/storage/volume-snapshot-classes/) to use for volume snapshotting.
+
+
+Let's create the `BackupConfiguration` crd we have shown above.
+
+
+
+
+```console
+$ kubectl apply -f ./docs/examples/volume-snapshot/backupconfiguration.yaml
+backupconfiguration.stash.appscode.com/deployments-volume-snapshot created
+```
+here, `deployments-volume-snapshot` is the name of the `BackupConfiguration` crd that has been created successfully.
+
+##### Verify Creating Volume Snapshot 
+
+If everything goes well, the Stash operator will create a `BackupSession` crd with the CronJob schedule. `BackupSession` crd name will be automatically generated by the following pattern `<BackupConfiguration name>-<creation timestamp in Unix epoch seconds>` and Label should be set as the following pattern `backup-configuration:<BackupConfiguration name>`.
+
+Now, wait a few minutes,  you will see that `BackupSession` has been created.
+Check that the phase of `BackupSession` status is `Running` 
+
+```console
+$ kubectl get bs -n demo -l  backup-configuration=deployments-volume-snapshot
+NAME                                     BACKUPCONFIGURATION           PHASE       AGE
+deployments-volume-snapshot-1559026686   deployments-volume-snapshot   Running   26s
+```
+here, `deployments-volume-snapshot-1559026686` is the name of the `BackupSession` crd that has been created successfully after 1 minute.
+
+
+`BackupSession` crd will create volume snapshotter job, job name follows the following pattern `volume-snapshot-<BackupSession name>`. 
+Check it by using the following command
+
+
+```console
+$ kubectl get job -n demo volume-snapshot-deployments-volume-snapshot-1559026686
+NAMESPACE   NAME                                                     COMPLETIONS   DURATION   AGE
+demo        volume-snapshot-deployments-volume-snapshot-1559026686   1/1           19s        40s
+```
+
+Finally, snapshotter job will create volume snapshot vai Kubernetes native API. `VolumeSnapshot` name will be formed by the following pattern `<PVC name>-<backup session creation timestamp in Unix epoch seconds>`.
+
+Wait a few minutes,  you will see that `backup-pvc-1559026686` Volume Snapshot has been created Successfully.
+
+```console
+$ kubectl get volumesnapshot -n demo backup-pvc-1559026686
+NAMESPACE   NAME                           AGE
+demo        backup-pvc-1559026686          153m
+
+```
+If you check the status of the `BackupSession` crd then we will see that the phase of backupsession status is `Succeeded`. 
```

Comment:
Once the snapshotting is completed, BackupSession crd phase will be `Succeeded`. Check the `BackupSession` phase is `Succeeded` by following command: